ballerina-tests/tests/http_client_url_encoded_content_test.bal

```diff
@@ -46,6 +46,17 @@ isolated function testUrlContentWithPost() returns error? {
     test:assertEquals(response, expectedResponse, msg = "Found unexpected output");
 }
 
+@test:Config {
+    groups: ["urlEncodedContent"]
+}
+isolated function testUrlContentInline() returns error? {
+    string response = check clientUrlEncodedTestClient->post("", {
+        "key1": "value1",
+        "key2": "value2"
+    }, mediaType = mime:APPLICATION_FORM_URLENCODED);
```

Comment:
Let's properly format this.

Reply:
```ballerina
string response = check clientUrlEncodedTestClient->post("", 
  {
    "key1": "value1",
    "key2": "value2"
  }, 
  mediaType = mime:APPLICATION_FORM_URLENCODED
);
```

Reply:
Fixed with [e68158206c5ae3747c86e5af80f533ea2b987e18](https://github.com/ballerina-platform/module-ballerina-http/pull/839/commits/e68158206c5ae3747c86e5af80f533ea2b987e18)